.python-version

```diff
@@ -0,0 +1 @@
+3.8.20
```

Comment:
Don't we just want `3.8` instead?

Note for others: this version is used by `uv venv`, but not by `uv sync`

Reply:
> > Don't we just want `3.8` instead?
> 
> You mean that the strict python version could be defined in the lock file? I haven't checked that.

It does not seem to be set there (that puzzles me a bit)

> > Note for others: this version is used by `uv venv`, but not by `uv sync`
> 
> It is used by both

I mean that changing/removing that file and running `uv sync` does not change the interpreter in the venv, whereas `uv venv` does.

Reply:
Maybe I drew conclusions a bit too fast, but nevertheless:

```
(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ uv run python --version
Python 3.8.20
(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ rm .python-version 
(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ uv sync
Resolved 53 packages in 0.89ms
Audited 38 packages in 0.03ms
(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ uv run python --version
Python 3.8.20

(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ uv venv
Using CPython 3.10.18
Creating virtual environment at: .venv
Activate with: source .venv/bin/activate
(tests) user@work-vates:tests (gln/uv-setup-tests-uzqs =)$ uv run python --version
Installed 38 packages in 263ms
Python 3.10.18
```

Reply:
We want to pin a version, in order to create an environment as reproducible as possible.
Note that it doesn't prevent us to use another python version, with or without uv, as now documented in the readme

Reply:
The question is, what extra level of reproducibility do you expect from pinning to maintenance releases?  It's not like a compiler where we care about bit-by-bit reproducibility of build artifacts, or like a checker that constantly adds more checks. What problems could arise from using the latest maintenance release of a supported Python version?  Here we clearly add more work work on our side just to get the latest fixes when they are out (and we won't want to do extensive non-ref testing anyway when we do so): to me it just looks like the extra burden is not worth the gains.